Code/Framework/AzCore/AzCore/Math/Geometry3DUtils.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Math/Geometry3DUtils.h>
+#include <AzCore/std/function/function_template.h>
+#include <AzCore/Casting/numeric_cast.h>
+
+namespace AZ::Geometry3DUtils
```

Comment:
uber nit: I believe per the coding standards this should be `Geometry3dUtils` to be more consistent with our other naming conventions

Reply:
Fixed! I had aligned it with the existing "Geometry2DUtils", but since this is a new file, it seems reasonable to switch conventions as well. I'm leaving the name of the file itself as "Geometry3DUtils" to avoid any issues that can arise from renaming a file with only a case change.

Reply:
Ah cool yes sorry I spotted this afterwards 😅 thanks for making the updates! 🙂 Good call on the rename, seems like the safest option 😅